apps/cms/src/components/question/question.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
We will use question.choice-question and question.text-answer-question, please remove it

Reply:
updated